README.md

```diff
@@ -197,10 +197,12 @@ Lit.store_request_info = true
 
 ### Testing
 
-For local testing [Appraisal](https://github.com/thoughtbot/appraisal) gem comes into play, run tests via: `bundle exec appraisal rails-4.2 rake test`.
-
-Please remember to edit `test/dummy/config/database.yml` file
+1. gem install bundler appraisal                    # Install some gems
```

Comment:
With `Gemfile` checked in we'll just need bundler and then we can do `bundle exec appraisal <whatever commands>`

Reply:
I'd prefer that too, but it broke the suite.  Maybe merge this PR and you guys give that a shot?